core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -575,7 +575,7 @@ class KafkaServer(
   protected def createRemoteLogManager(config: KafkaConfig): Option[RemoteLogManager] = {
     val remoteLogManagerConfig = new RemoteLogManagerConfig(config)
     if (remoteLogManagerConfig.enableRemoteStorageSystem()) {
-      if(config.logDirs.size > 1) {
+      if(config.logDirs.lengthCompare(1) > 0) {
```

Comment:
FYI reviewers -
config.logDirs is a scala Seq, hence, lengthCompare() is beneficial.